### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by VeSpy4u**

### CODE REVIEW

File: Tasks/except.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-   [].sort(() => 2000);
-if (no.includes(Z)) { delete incomingvaluesarray[Z]; return;} else { return; delete incomingvaluesarray[Z]; }
-  }, 2000);
-  ({ key: 'value' });
-  return incomingvaluesarray}
+const except = (obj, ...excision) => {
+  for (const key of Object.keys(obj)) {
+    if (excision.includes(key)) delete obj[key];
+  }
+  return obj;
```
Comment (tshemsedinov) [inline Tasks/except.js:8]: Don't change incoming parameters

File: Tasks/merge.js [hunk 2]
```diff
@@ -1,19 +1,6 @@
...
-       );
-        object_3[attribute_name] = object_1[attribute_name]
-  }
-   for (attribute_name in object_2) {
-      (object_3[attribute_name] == object_2[attribute_name])
-     object_3[attribute_name] = object_2[attribute_name]
-    }
-return object_1, object_2, object_3;
-})
+const merge = (objOne, objTwo) => Object.assign({}, objOne, objTwo);
```
Comment (tshemsedinov) [inline Tasks/merge.js:4]: Implement this with for loops, don't use Object.assign

Comment (VeSpy4u) [reply Tasks/merge.js:4]: https://github.com/HowProgrammingWorks/Exams/pull/61/commits/7d5678bdb71c0625b62a2e410f942dbd6817cb50

File: Tasks/except.js (hunk 1 above)
Comment (VeSpy4u) [reply Tasks/except.js]: https://github.com/HowProgrammingWorks/Exams/pull/61/commits/d9df74fe8c08b98d94364b5057c0f5740e23b5b2

File: Tasks/merge.js (hunk 2 above)
Comment (VeSpy4u) [reply Tasks/merge.js]: https://github.com/VeSpy4u/Exams/commit/6ea39056ddc91ac9db331989a2f24de6fc2a50e5#diff-583e7a1e779616b1571de750b4b0efa988eb1059b763e3fa4402f57a61ed9609

Comment (VeSpy4u) [reply Tasks/merge.js]: https://github.com/VeSpy4u/Exams/commit/6ea39056ddc91ac9db331989a2f24de6fc2a50e5

**Pull request closed without merging**